src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -523,28 +527,17 @@ void VideoDecoder::addVideoStreamDecoder(
       videoStreamOptions.colorConversionLibrary.value_or(defaultLibrary);
 }
 
-void VideoDecoder::updateMetadataWithCodecContext(
```

Comment:
I inlined this above, as it became hard to justify this being a single separate method with only one call-site.